site/src/common/helpers/SvgUse.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+interface SvgUseProps extends React.SVGProps<SVGSVGElement> {
+    xlinkHref: string;
+}
+
+export const SvgUse: React.FunctionComponent<SvgUseProps> = ({ xlinkHref, ...props }) => (
+    <svg {...props}>
+        <use xlinkHref={createAssetUrl(xlinkHref)} />
+    </svg>
+);
```

Comment:
Why are we using xlinkHref although it's deprecated (https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href)? 
Support of xlinkHref is only ~ 0.7% better (https://caniuse.com/?search=use%20href).

MDN states
> If you need to support earlier browser versions, the deprecated xlink:href attribute can be used as a fallback in addition to the href attribute, e.g. <use href="some-id" xlink:href="some-id" x="5" y="5" />.

So we could do

```suggestion
interface SvgUseProps extends React.SVGProps<SVGSVGElement> {
    href: string;
}

export const SvgUse: React.FunctionComponent<SvgUseProps> = ({ href, ...props }) => {
    const assetUrl = createAssetUrl(href);
    return (
        <svg {...props}>
            <use href={assetUrl} xlinkHref={assetUrl} />
        </svg>
    );
};
```

Reply:
Nice catch! 👏🏼 We should use your suggestion.